src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
The properties `a`, `b`, and `c` are declared in the constructor but are not initialized as class properties. You need to declare them as properties of the `Triangle` class to use them in other methods like `getArea`.